.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/web-monitoring-db
+    docker:
+      - image: circleci/ruby:2.4.1-node
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: root
+          HOST_URL: 'web-monitoring-db.test'
+          ALLOWED_ARCHIVE_HOSTS: 'https://edgi-versionista-archive.s3.amazonaws.com/ https://test-bucket.s3.amazonaws.com/'
+      - image: circleci/postgres:9.5-alpine
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: circle-test_test
```

Comment:
Do we actually need either of these environment vars for setting up PG? Since we are manually running `rake db:create` below, we shouldn’t need to tell PG what default DB to create with this env var. Ditto, I think, for the user.

Similarly, do we need them set in Ruby container’s env above? (`PGHOST` and `PGUSER`) Genuinely curious to know since I haven’t tried to do 2.0 on any projects yet.

Reply:
In CircleCI 2.0, it's docker based. Not having these lines will cause Rails to attempt running on a unix socket, and fail.

Reply:
Well 💩! The [Ruby language guide](https://circleci.com/docs/2.0/language-ruby/) made me hope that wasn't the case.